vignettes/model_definition.Rmd

```diff
@@ -26,32 +26,32 @@ The `baselinenowcast` model is based on the reference model for the COVID-19 hos
 We denote $X_{t,d}, d = 0, \dots, D$ as the number of cases occurring at time $t$ which appear in the dataset with a delay of $d$. For example, a delay $d = 0$ means that a case occurring on day $t$ arrived in the dataset on day $t$. We only consider cases reporting within a maximum delay $D$. The number of cases reporting for time $t$ with a delay of at most $d$ can be written as:
 
 $$
-X_{t, \le d} = \sum_{i=0}^d X_{t,i}
+X_{t, \le d} = \sum_{i=0}^d X_{t,i} (\#eq:Xltd)
```

Comment:
Am copying the syntax from primarycensoreddist... https://github.com/epinowcast/primarycensored/blob/064b1eb656b47e944aa0d3471d08069a0bf56688/vignettes/why-it-works.Rmd#L95

Reply:
But this prints like which is obviously not what we want
<img width="354" height="107" alt="Screenshot 2025-09-08 at 17 51 38" src="https://github.com/user-attachments/assets/5e569058-5638-48a5-a0bd-c92a530488df" />